fontbox/src/main/java/org/apache/fontbox/cff/Type1CharString.java

```diff
@@ -161,15 +159,15 @@ private void render()
         });
     }
 
-    private List<Number> handleType1Command(List<Number> numbers, CharStringCommand command)
+    private void handleType1Command(List<Number> numbers, CharStringCommand command)
     {
         commandCount++;
         Type1KeyWord type1KeyWord = command.getType1KeyWord();
         if (type1KeyWord == null)
         {
             // indicates an invalid charstring
             LOG.warn("Unknown charstring command in glyph {} of font {}", glyphName, fontName);
-            return Collections.emptyList();
+            numbers.clear();
```

Comment:
The immediate return is missing, so that the switch runs into the default case and throws an exception

Reply:
Sorry for the mistake! We push this patch.